bridges/opentracing/lib/opentelemetry/bridge/opentracing/scope_manager.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # A ScopeManager provides an API for interfacing with
+      # OpenTelemetry Tracers and Spans as OpenTracing objects
+      class ScopeManager
+        SCOPE_KEY = :__opentelemetry_opentracing_scope__
+        MANAGER_KEY = :__opentelemetry_opentracing_scope_manager__
+        private_constant :SCOPE_KEY
+        private_constant :MANAGER_KEY
+
+        def self.current
```

Comment:
I don't think we need this at all? I think we just need `Tracer#scope_manager` to return the singleton instance.